pkg/imgutil/pull/pull.go

```diff
@@ -73,7 +73,8 @@ func Pull(ctx context.Context, client *containerd.Client, ref string, config *Co
 	opts := []containerd.RemoteOpt{
 		containerd.WithResolver(config.Resolver),
 		containerd.WithImageHandler(h),
-		containerd.WithSchema1Conversion,
+		//nolint:staticcheck
+		containerd.WithSchema1Conversion, //lint:ignore SA1019 trying out line-based linter directives
```

Comment:
?

Reply:
Sorry! I was not sure how to use lint:ignore, so that was a stranded comment, I have fixed it now. We need both ignore to ignore on local runs and also golangci